connector/mqtt/vernemq/broker/bin/vmq_dojot/scripts_tls/retrieveCRL.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
For now we can keep this way, but I think that EJBCA has a public http/s endpoint to get the CRL